### ISSUE TITLE
Build and publish 'Educational and career counseling (VA Chapter 36)' page in Spanish

### ISSUE THREAD

**Issue by moviedosoco** [department-of-veterans-affairs/va.gov-team#85988]
## Issue description
We need to build this page in Drupal, review it, and publish it.

-https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/translation-work/State%20Department%20translation%20work/careers%20and%20employment/Educational%20and%20career%20counseling%20(VA%20Chapter%2036).docx

---
## Tasks
- [ ] Build page in Drupal (Victoria)
- [ ] Review (Sara)
- [ ] Publish (Sara)
- [ ] Send live page to State Dept. for QA (Randi)
- [ ] Input any changes (Victoria)
- [ ] Review (Sara)
- [ ] Publish with Spanish toggle (Sara)

## Acceptance Criteria
- [ ] Page is live

## Configure this issue
- [ ] #75574

**Comment by RLHecht** [comment]
@moviedosoco Here's the translation back from State. This page is ready to be built in Drupal now.

[SPA-Educational and career counseling (VA Chapter 36).docx](https://github.com/user-attachments/files/16234828/SPA-Educational.and.career.counseling.VA.Chapter.36.docx)

**Comment by moviedosoco** [comment]
@saratorres2 This page is ready to review and publish: "Asesoramiento educativo y profesional (Capítulo 36 del VA)"

**Comment by moviedosoco** [comment]
Note that I replaced the Spanish contraction "del" with the preposition "de" in sentences containing the phrase "del VA," as decided at our translation meeting.

**Comment by saratorres2** [comment]
This page is now published and live. https://www.va.gov/asesoramiento-educativo-y-profesional-capitulo-36-de-va

**Comment by moviedosoco** [comment]
@saratorres2 Thank you, Sara!

**Comment by moviedosoco** [comment]
@saratorres2 I added the "_(en inglés)_" for the _Preparación y empleo para veteranos (VR&E)_ text link, in section _Otra información_. The page is ready for republishing. Thank you.

**Comment by saratorres2** [comment]
@moviedosoco I've published the page with your update!

**Comment by moviedosoco** [comment]
@saratorres2  Thank you!

**Comment by moviedosoco** [comment]
@RLHecht @saratorres2 The Spanish page was sent for QA on 9/9/24: https://www.va.gov/careers-employment/education-and-career-counseling-esp/

**Comment by RLHecht** [comment]
@moviedosoco here's the QA back from State:

[LS-2024-0122945-LQA Notes-es_v1__Educational and career counseling (VA Chapter 36).docx](https://github.com/user-attachments/files/17131167/LS-2024-0122945-LQA.Notes-es_v1__Educational.and.career.counseling.VA.Chapter.36.docx)

**Comment by moviedosoco** [comment]
@RLHecht Changes were input in Drupal.

@saratorres2 The page _Asesoramiento educativo y profesional (Capítulo 36 de VA_) is ready for review and publish with the Spanish toggle:

http://preview-prod.vfs.va.gov/preview?nodeId=70645

**Comment by RLHecht** [comment]
@juliana-jara can you review the Drupal page as well as compare it to the word doc that came back from State? If everything looks good I can publish and will create the ticket to get the toggle added. Thanks!

**Comment by juliana-jara** [comment]
@RLHecht 
There are two moments when the content in Spanish doesn't match the live page in English. I prepared the document for State in [this Sharepoint file](https://dvagov.sharepoint.com/:w:/r/sites/oitcontentiateam/Shared%20Documents/Translation/Ticket%20%2385988/Educational%20and%20career%20counseling%20(VA%20Chapter%2036).docx?d=w55315c66ac1f4e3a87c3c0005207c77c&csf=1&web=1&e=mqzE8Q). Here are the details of what needs to be updated:

- This paragraph needs to be added to the Spanish version. 

<img width="1476" height="552" alt="Image" src="https://github.com/user-attachments/assets/2e3f86bb-da94-483a-b5c8-2228500dceec" />

- Additionally, this section needs to be fully rewritten.

<img width="1508" height="982" alt="Image" src="https://github.com/user-attachments/assets/d72a1cc2-4f6b-4c74-89f5-206c237e7a04" />

**Comment by RLHecht** [comment]
Thanks! I just sent this to State.

**Comment by RLHecht** [comment]
@juliana-jara here is this content back from State. Since we had previously published this page we can continue moving this one along.

[Educational and career counseling (VA Chapter 36) new content-SPA-FINAL.docx](https://github.com/user-attachments/files/21619184/Educational.and.career.counseling.VA.Chapter.36.new.content-SPA-FINAL.docx)

**Comment by RLHecht** [comment]
I think if we have the form number then translating it into Spanish is fine without writing it in English as well. But I am good with you setting the style for this and what you think makes sense. And given that we are pausing a lot of this work, this isn't a rush!

**Comment by juliana-jara** [comment]
@RLHecht Absolutely! I just updated that on Drupal https://prod.cms.va.gov/node/70645/latest
